charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
+metadata:
+  name: etcd-{{ .Values.role }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    role: {{ .Values.role }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.podAnnotations }}
+  annotations:
+{{ toYaml .Values.podAnnotations | indent 4 }}
+{{- end }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    app: etcd-statefulset
+    role: {{ .Values.role }}
+    networking.gardener.cloud/to-dns: allowed
+    networking.gardener.cloud/to-public-networks: allowed
+    networking.gardener.cloud/to-private-networks: allowed
+  etcd:
+    tls:
+      serverTLSSecretRef:
+        name: {{ .Values.tlsServerSecretName }}
+        namespace: {{ .Release.Namespace }}
+      clientTLSSecretRef:
+        name: {{ .Values.tlsClientSecretName }}
+        namespace: {{ .Release.Namespace }}
+    metrics: {{ .Values.metrics }}
+    version: {{ .Values.etcdVersion }}
+    defragmentationSchedule: '0 */24 * * *'
+{{- if .Values.etcdResources }}
+    resources:
+{{ toYaml .Values.etcdResources | indent 6 }}
+{{- end }}
+    clientPort: {{ .Values.servicePorts.client }}
+    serverPort: {{ .Values.servicePorts.server }}
+  backup:
+    version: {{ .Values.brVersion }}
```

Comment:
This is only the image tag, but where is the image repository specified?

Reply:
I have kept the repository to be not configurable. Only the tags can be updated. The repository information is fetched from [here](https://github.com/gardener/etcd-druid/blob/master/charts/images.yaml).

Reply:
Same as https://github.com/gardener/gardener/pull/1762#discussion_r363212982